lib/server.js

```diff
@@ -0,0 +1,74 @@
+/* @flow */
+
+import tmp from 'tmp'
+import Path from 'path'
+import { async as getAsyncEnv } from 'consistent-env'
+import { BufferedProcess, Disposable, CompositeDisposable } from 'atom'
+import type { Server } from './types'
+
+const JULIA_SERVER_PATH = Path.join(__dirname, 'julia-server.jl')
+
+export async function getPipePath(): Promise<{ subscription: CompositeDisposable, path: string }> {
+  const subscription = new CompositeDisposable()
+  let path = await new Promise(function(resolve, reject) {
+    tmp.file(function(error, tempPath, fd, cleanupCallback) {
```

Comment:
I guess tmp.file creates a temporary file. Unfortunately this won't work, because the lintserver() wants to create the file and cannot, if the file already exists:
```julia
julia> a = "/tmp/tmp-17077g9zhwjrfaehc.tmp"
"/tmp/tmp-17077g9zhwjrfaehc.tmp"

shell> ls $a
ls: cannot access /tmp/tmp-17077g9zhwjrfaehc.tmp: No such file or directory

shell> touch $a

shell> ls $a
/tmp/tmp-17077g9zhwjrfaehc.tmp

julia> lintserver(a,"standard-linter-v2")
ERROR: ArgumentError: could not listen on path /tmp/tmp-17077g9zhwjrfaehc.tmp
 in listen(::String) at ./stream.jl:944
 in lintserver(::String, ::String) at /home/tero/.julia/v0.5/Lint/src/Lint.jl:468

shell> rm $a

julia> lintserver(a,"standard-linter-v2")
Server running on port/pipe /tmp/tmp-17077g9zhwjrfaehc.tmp ...
^CServer closed

shell> ls $a
ls: cannot access /tmp/tmp-17077g9zhwjrfaehc.tmp: No such file or directory
```
It looks that if lintserver closes properly it will clean the temporary file as well.